pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -734,7 +734,7 @@ func (gc *GarbageCollector) processAttemptToOrphanWorker(logger klog.Logger) boo
 func (gc *GarbageCollector) attemptToOrphanWorker(logger klog.Logger, item interface{}) workQueueItemAction {
 	owner, ok := item.(*node)
 	if !ok {
-		utilruntime.HandleError(fmt.Errorf("expect *node, got %#v", item))
+		utilruntime.HandleErrorWithLogger(logger, fmt.Errorf("expect *node, got %#v", item), "Invalid item in attemptToOrphanWorker")
```

Comment:
And here.

Usually `%T` would be used to show the actual type, not the variable content. I think it's okay to update this when touching the line.